src/validation.cpp

```diff
@@ -805,9 +803,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     // If we don't hold the lock allConflicting might be incomplete; the
     // subsequent RemoveStaged() and addUnchecked() calls don't guarantee
     // mempool consistency for us.
-    fReplacementTransaction = setConflicts.size();
-    if (fReplacementTransaction)
-    {
+    if (setConflicts.size()) {
```

Comment:
nit: Not sure if we have this in the coding guidelines, but personally I'm not a fan of implicit int-to-bool casts. Could avoid it via e.g.
```suggestion
    if (setConflicts.size() > 0) {
```
or
```suggestion
    if (!setConflicts.empty()) {
```